### PR TITLE
Added isStagePixel and curStage for if statements

### PR DESCRIPTION
**Pull request by rodney528**

### CODE REVIEW

**Comment by ImaginationSuperHero528** [comment]
For Lua... cause why not?

**Review by rodney528 — COMMENTED** [review]
Now to wait for it to be accpected or declined.

File: source/FunkinLua.hx [hunk 1]
```diff
@@ -189,6 +189,8 @@ class FunkinLua {
 		set('noResetButton', ClientPrefs.noReset);
 		set('lowQuality', ClientPrefs.lowQuality);
 		set("scriptName", scriptName);
+		set('isPixelStage', PlayState.isPixelStage); // BTW idrk if these will actually work properly since I don't fully know what I'm doing with haxe lmao I only have some understandings like how I had to add the "PlayState." part
+		set('isPixelStage', PlayState.curStage);
```
Comment (frantastic24) [inline source/FunkinLua.hx:193]: What

**Comment by frantastic24** [comment]
I don't think this is necessary tbh

**Comment by CrusherMods** [comment]
can't you just use
```lua
if getProperty('isPixelStage') then
```
???

**Comment by frantastic24** [comment]
You can, which is why I don't think this is necessary or even useful since both variables are very rarely used

**Comment by Mastercopper2346** [comment]
You forgot to change isPixelStage to curStage

**Comment by ImaginationSuperHero528** [comment]
> You forgot to change isPixelStage to curStage

Wdym I meant for both to be there?

**Comment by CrusherMods** [comment]
> > You forgot to change isPixelStage to curStage
> 
> Wdym I meant for both to be there?

image

File: source/FunkinLua.hx (hunk 1 above)
Comment (Raltyro) [reply source/FunkinLua.hx:193]: what??

**Comment by ImaginationSuperHero528** [comment]
Fixed an error opss!

**Review by DabbinDude316B — APPROVED** [review]

**Comment by An-undertale-fan** [comment]
If curStage is kinda useless since you can write the code in the stage lua :v

**Comment by ImaginationSuperHero528** [comment]
Changed to a custom branch.

**Comment by ImaginationSuperHero528** [comment]
Not dead go to #9350

**Pull request closed without merging**